api/app/hourlies.py

```diff
@@ -57,15 +57,26 @@ async def fetch_hourly_readings_from_db(
     return result
 
 
+def _get_five_day_interval(time_of_interest: datetime):
+    five_days_past = time_of_interest - timedelta(days=5)
+
+    # Prepare query params and query:
+    start_time_stamp = math.floor(five_days_past.timestamp()*1000)
+    end_time_stamp = math.floor(time_of_interest.timestamp()*1000)
+
+    return start_time_stamp, end_time_stamp
+
+
 async def get_hourly_readings(
         station_codes: List[int],
         time_of_interest: datetime) -> List[WeatherStationHourlyReadings]:
     """ Get the hourly readings for the list of station codes provided.
     Reading 5 days before, and 10 days after the time of interest are returned.
     Depending on configuration, will read from WF1 or from local database.
     """
+    start_time_stamp, end_time_stamp = _get_five_day_interval(time_of_interest)
```

Comment:
This behaviour is inconsistent with the method comments and what fetch_hourly_readings_from_db is doing (it's grabbing 5 days past and 10 days future)